test/Transforms/Flattening/mastermind.llzk

```diff
@@ -0,0 +1,3393 @@
+// RUN: llzk-opt %s -llzk-flatten -llzk-array-to-scalar
+// XFAIL:*
```

Comment:
Is addressing this XFAIL a task for this PR? Also, should we just keep one copy of `mastermind.llzk` in the `Inputs` directory and enable different tests via `include` inlining?

Reply:
Yes. That's a good idea.